commands

```diff
@@ -45,7 +45,7 @@ case "$1" in
     docker run --rm --link "$SERVICE_NAME:$PLUGIN_COMMAND_PREFIX" dokkupaas/wait > /dev/null
 
     dokku_log_verbose_quiet "Creating container database"
-    DATABASE_NAME="$(get_database_name "$SERVICE")"
+    DATABASE_NAME="yesware_next_development"
```

Comment:
I'm not sure this is necessary, and again prevents us from merging back to the original repo.  This only creates the main app's database - all the others use different ones.  I think most people expect to run `rake db:create` for a new app anyway.